staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/envelope.go

```diff
@@ -132,7 +139,11 @@ func (t *envelopeTransformer) TransformToStorage(data []byte, context value.Cont
 	})
 	b.AddBytes(result)
 
-	return b.Bytes()
+	out, err := b.Bytes()
+	if err != nil {
+		return nil, status.Error("failed to get encrypted data")
```

Comment:
why are we masking the returned error here?

Reply:
should be `status.Error(codes.Internal, "failed to get encrypted data")`